lib/rucio/core/transfer.py

```diff
@@ -662,13 +662,13 @@ def get_transfer_requests_and_source_replicas(process=None, total_processes=None
                 fts_list = fts_hosts.split(",")
 
                 verify_checksum = 'both'
-                if rse_attrs[dest_rse_id].get('verify_checksum', 'True').lower() == 'false':
-                    if rse_attrs[source_rse_id].get('verify_checksum', 'True').lower() == 'false':
+                if not rse_attrs[dest_rse_id].get('verify_checksum', True):
```

Comment:
is there a reason why it's not

if rse_attrs.get(...   , False):

?

Reply:
Because if the attribute is not set, we still want checksum checks (per default)?